docs/maatregelen/neem_technische_documentatie_in_algoritmeregister.md

```diff
@@ -0,0 +1,45 @@
+---
+# vul hier een titel in voor deze maatregel
+title: Neem technische documentatie op in het algoritmeregister
+# geef hier een korte toelichting van deze maatregel
+toelichting: Neem geschikte informatie uit technische documentatie op in het algoritmeregister
+# vul hier de bestandsnamen in van de vereisten die horen bij deze maatregel
+vereiste: algoritmeregister
+# vul hier de fasen van de levenscyclus in die horen bij deze maatregel
+levenscyclus: 
+- ontwikkel
+- verificatie-en-validatie
+- implementatie
+- monitoring-en-beheer
+# vul hier de bouwblokken in die horen bij deze maatregel
+bouwblok:
+- transparantie
+rollen:
+- proceseigenaar
+- privacy officer
+- data scientist
+- data engineer
+- beleidsmedewerker
+---
+
+<!-- Let op! onderstaande regel met 'tags' niet weghalen! Deze maakt automatisch de knopjes op basis van de metadata  -->
+<!-- tags -->
+
+## Maatregel
+<!-- Vul hier een omschrijving in van wat deze maatregel inhoudt. -->
+Neem geschikte informatie uit technische documentatie op in het algoritmeregister.
+
+## Toelichting 
+<!-- Geef hier een toelichting van deze maatregel -->
+Met het opstellen van technische documentatie over algoritmes en AI-systemen wordt belangrijke informatie vastlegd die moet worden opgenomen of kan worden opgenomen in het algoritmeregister. De Handreiking Algoritmeregister kan worden geraadpleegd om te bepalen of het algoritme of AI-systeem moet worden gepubliceerd. 
```

Comment:
Kunnen we dit concreter maken? Welke informatie bedoelen we dan? Is het ook nog nuttig hierbij te verwijzen naar de publicatiestandaard van het register? https://algoritmes.pleio.nl/attachment/entity/a47f2708-48bd-4a10-8893-ab24ac8c7207

Reply:
Goed voorstel, ik voeg het toe! Wel ervoor gekozen om in het midden te laten om welke informatie het exact gaat. Dat lijkt me iets wat organisaties zelf moeten onderzoeken.